src/serve-remix-response.ts

```diff
@@ -27,8 +36,17 @@ export async function serveRemixResponse(
     values.push(value)
   }
 
+  if (response.body instanceof ReadableStream) {
+    return {
+      // @ts-expect-error
+      data: Readable.from(response.body),
```

Comment:
Don't like this here, but it works. `Readable.from` expects an `Iterable` and response.body is already a `ReadableStream<UInt8Array>`, but only in this constellation it works.